### PR TITLE
Presentation: Avoid loading assets in presentation-common esm module

### PR DESCRIPTION
**Pull request by saskliutas**
Moved getLocalizedStringEN function to `presentation-backend` as this is the only place it is used. This change removes requirement to copy localization assets when building `presentation-common` esm module.

### CODE REVIEW

File: presentation/backend/src/presentation-backend/Utils.ts [hunk 1]
```diff
@@ -11,6 +11,21 @@ import { Element, IModelDb } from "@itwin/core-backend";
 import { DbResult, Id64String } from "@itwin/core-bentley";
 import { Diagnostics, DiagnosticsOptions, InstanceKey } from "@itwin/presentation-common";
 
+const ecPresentation = require("@itwin/presentation-common/assets/locales/en/ECPresentation.json"); // eslint-disable-line @typescript-eslint/no-var-requires
```
Comment (saskliutas) [inline presentation/backend/src/presentation-backend/Utils.ts:14]: @aruniverse tried using `require("@itwin/presentation-common/lib/assets/locales/en/ECPresentation.json")` but it didn't work.

`require("@itwin/presentation-common/lib/cjs/assets/locales/en/ECPresentation.json")` seems to work. Should I use it instead?

Comment (aruniverse) [reply presentation/backend/src/presentation-backend/Utils.ts:14]: I think thats expected. I believe you'd need to update the `build:assets` script in presentation-common to copy to `lib/assets` not `lib/cjs/assets`

**Comment by aruniverse** [comment]
Just to confirm, you mean to merge this change into `imodel02` not `master`?

**Review by miniant-git — APPROVED** [review]

**Review by roluk — APPROVED** [review]

**Comment by saskliutas** [comment]
> Just to confirm, you mean to merge this change into `imodel02` not `master`?

Yes. Localizations changes that caused this are still only in `imodel02`

File: presentation/backend/src/presentation-backend/Utils.ts (hunk 1 above)
Comment (saskliutas) [reply presentation/backend/src/presentation-backend/Utils.ts]: Ok. Changed to `require("@itwin/presentation-common/lib/cjs/assets/locales/en/ECPresentation.json")`

**Review by aruniverse — APPROVED** [review]

**Pull request merged**